doc/classes/GrooveJoint2D.xml

```diff
@@ -46,8 +46,10 @@
 	</methods>
 	<members>
 		<member name="initial_offset" type="float" setter="set_initial_offset" getter="get_initial_offset">
+			The grooves initial offset of the on body A.
```

Comment:
Needs a rewrite. It's the starting distance to the first body attached to the joint?